xml/article_sap_trento.xml

```diff
@@ -331,7 +331,7 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         <screen>HELM_EXPERIMENTAL_OCI=1 helm upgrade \
    --install trento-server \
    oci://registry.suse.com/trento/trento-server \
-   --version 0.4.3 \
+   --version 0.4.4 \
```

Comment:
Can we avoid the exact version? Or replace it with a placeholder? Or in other words: is there any command or document where a customer can look up this information? Maybe a command (`trento-server --version`) which can show this piece?

The reason for my questions is that we **should avoid** mentioning versions at all. It's just a maintenance nightmare. If we still need it now, that's okay for the time being. But in the long run, it should be replaced.

So my preferred suggestions would be to tell the customer to replace a `VERSION` placeholder with the current Trento (server) version.

Reply:
Thanks for your comments @tomschr . At the moment, there is no work around to indicate the exact version of the Trento helm chart and, unfortunately, the helm chart version is not aligned with the Trento version. I will talk with the Development Team to see if it's possible to include a "latest" flag. That would prevent us from having to update the installation instructions every time a new version is published and would avoid confusion with the Trento version.

Reply:
That would the best idea! :+1: I like that. Thanks Alberto. :+1: